Tevinthuku/Politico
# #163908188 reset password endpoint

**What does this PR do?**
This PR adds an endpoint where a user can post their email if they need to reset their password 
via the `/api/v2/auth/reset` endpoint.

**Description of Task to be completed?**
- [x] Create tests for the `/api/v2/auth/reset` endpoint
- [x] Create the endpoint for a password reset.

**How should this be manually tested?**
1. Clone this repo
2. Follow readme on the setup instructions.
3. Run the tests via the `python -m pytest --cov=app/api` command
4. Open postman and play with the endpoint. (See screenshots below)


**What are the relevant pivotal tracker stories?**
[#163908188](https://www.pivotaltracker.com/n/projects/2241721/stories/163908188)

**Screenshots**
![image](https://user-images.githubusercontent.com/12128153/52663817-8777b280-2f18-11e9-80a8-e99b92ab630a.png)

![image](https://user-images.githubusercontent.com/12128153/52663855-a118fa00-2f18-11e9-8dec-7bc280d1d243.png)

![image](https://user-images.githubusercontent.com/12128153/52663972-f48b4800-2f18-11e9-908e-b4c8be27185e.png)
